sdk/workflow-tengo/src/workdir/index.lib.tengo

```diff
@@ -16,9 +18,28 @@ enum := import("enum")
 validation := import(":validation")
 render := import(":render")
 
+_WD_CREATE_FIELD_ALLOCATION := "allocation"
+_WD_CREATE_FIELD_WORKDIR    := "workdir"
+
+_WD_FILL_FIELD_WORKDIR_IN   := "workdirIn"
+_WD_FILL_FIELD_WORKDIR_OUT  := "workdirOut"
+
+/**
+ * Create empty working directory
+ *
+ * @return smart.field - field where working directory will appear after creation
+ */
 create := func() {
-	wd := smart.ephemeralBuilder(constants.RTYPE_WORKDIR_CREATE).lockAndBuild()
-	return wd.getField("workdir")
+	// working directories without quotas (old-style)
+	wdCreate := smart.ephemeralBuilder(constants.RTYPE_WORKDIR_CREATE).lockAndBuild()
+	return wdCreate.getField(_WD_CREATE_FIELD_WORKDIR)
+}
+
+createV2 := func(allocation) {
```

Comment:
`allocation` is a reference to the storage space request, right? Let's put a comment here.

Reply:
Fixed.